### PR TITLE
Add JAMstack Berlin event

### PR DESCRIPTION
**Pull request by Khaledgarbaya**
Hey, Beautiful people, I would like to add my meetup JAMstack_Berlin to the list

Cheers,
Khaled

### CODE REVIEW

File: _posts/2019-05-27-jamstackberlin.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+layout: post
+title: JAMstack_Berlin
+date: 2019-05-27 18:00:00 UTC+2
+venue: Contentful foobar
```
Comment (komaeda) [inline _posts/2019-05-27-jamstackberlin.md:5]: Is the foobar here intentional?

File: _posts/2019-05-27-jamstackberlin.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+time: 6PM - 9PM
+href: https://www.meetup.com/JAMStack_berlin/events/260819537/
+---
```
Comment (komaeda) [inline _posts/2019-05-27-jamstackberlin.md:9]: Could you add a `ticket` property?

File: _posts/2019-05-27-jamstackberlin.md (hunk 1 above)
Comment (Khaledgarbaya) [reply _posts/2019-05-27-jamstackberlin.md:5]: Yep the venue is called foobar :D

**Comment by Khaledgarbaya** [comment]
@komaeda I addressed your changes, Thank you for the quick reaction

**Pull request merged**